README.md

```diff
@@ -1,13 +1,165 @@
 # OpenNMS Helm Charts
 
-OpenNMS Helm Charts makes it easier for users to run OpenNMS Horizon locally in a Kubernetes cluster.
-It is not yet available for a Cloud environment.
+If your organization uses Kubernetes or Red Hat OpenShift, OpenNMS makes a Helm chart available to simplify deployment of Horizon and Meridian.
 
 Each deployment through OpenNMS Helm Charts has a single Core server, Grafana, and a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.
 
 Note that this is one way to approach the solution.
 We recommend that you study the content of the Helm chart and tune it for your needs.
 
-## Charts
-* [Horizon](horizon/README.md)
-* [Minion](minion/README.md); This is an experimental chart.
+For more information about deploying in a containerized environment, including requirements and external dependencies, refer to [Containerized Deployment](https://docs.opennms.com/horizon/latest/deployment/core/containers.html) in the main product documentation.
+
+If you are not already familiar with Horizon or Meridian, we recommend reviewing our [product documentation](https://docs.opennms.com/horizon/32/deployment/core/introduction.html) and our [Horizon](https://www.opennms.com/horizon/) or [Meridian](https://www.opennms.com/meridian/) websites.
```

Comment:
Should we point the user to the Meridian documentation?

Reply:
I guess so. They are identical, but I can see why we would want to provide Meridian as well.